netbox_onboarding/onboarding_extensions/example_ios_set_device_role.py

```diff
@@ -0,0 +1,81 @@
+from netbox_onboarding.netbox_keeper import NetboxKeeper
```

Comment:
If this file is an example only, and isn't actually executed as code, consider moving it out of the source path and into a `docs/` directory?